app/src/main/java/com/github/se/project/ui/profile/CreateTutorProfile.kt

```diff
@@ -87,6 +88,20 @@ fun CreateTutorProfile(
                   style = MaterialTheme.typography.titleSmall,
                   modifier = Modifier.testTag("priceText"))
               PriceSlider(sliderValue)
+
+              // Description text field
+              OutlinedTextField(
+                  value = description,
+                  onValueChange = { description = it },
+                  modifier =
+                      Modifier.fillMaxWidth().padding(vertical = 4.dp).testTag("experienceField"),
+                  label = { Text("Do you have any experience as a tutor ?") },
+                  placeholder = {
+                    Text(
+                        "Share your previous tutoring experience, or if you don't have any, explain why you would be a great tutor.")
+                  },
+                  shape = MaterialTheme.shapes.medium,
+                  maxLines = 4)
             }
```

Comment:
The use of the OutlinedTextField for the experience input is clear and intuitive. The maxLines Property to 4 is a good idea to limit the space for input.